bootstrap/ansible/settings_template.tmpl

```diff
@@ -65,6 +65,33 @@ REQUEST_APPROVAL_CC_LIST = {{ request_approval_cc_list }}
 ALLOW_ALL_JOBS = True
 {% endif %}
 
+#------------------------------------------------------------------------------
+# File Storage Settings
+#------------------------------------------------------------------------------
+
+from gdstorage.storage import (GoogleDrivePermissionType,
+                               GoogleDrivePermissionRole,
+                               GoogleDriveFilePermission)
+                               
+GOOGLE_DRIVE_STORAGE_JSON_KEY_FILE = '{{ google_drive_private_key_file_path }}'
```

Comment:
Document that these settings are for the Google Drive module we're using (presumably), what each of them is, and that they can be ignored if the module is not being used.